eoscostarica/lifebank
# Sponsor signup and Styling app

## Sponsor signup and Styling app

### What does this PR do?
- Add Sponsor flow signup
- Add life bank theme
- Add UX/UI styling

### How should this be tested?
- run `make run`
- see the new look and feel
